cmake/SetupOptions.cmake

```diff
@@ -27,15 +27,20 @@ include(CMakeDependentOption)
 # Build options
 # Enable the tests only if this is the main project
 if(CMAKE_PROJECT_NAME STREQUAL "xad")
-    option(XAD_ENABLE_TESTS "Enable the XAD Tests" ON)
+    option(XAD_ENABLE_TESTS "Enable the XAD tests" ON)
 else()
     option(XAD_ENABLE_TESTS "Enable the XAD tests" OFF)
 endif()
 option(XAD_WARNINGS_PARANOID "Use extra-paranoid warning level" ON)
 option(XAD_POSITION_INDEPENDENT_CODE "Generate PIC code, so it can be linked into a shared library" ON)
 
-set(XAD_SIMD_OPTION "AVX" CACHE STRING "SIMD instruction set to use")
-set_property(CACHE XAD_SIMD_OPTION PROPERTY STRINGS SSE2 AVX AVX2 AVX512) # for drop-down in GUI
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "arm")
```

Comment:
Can we find out if this is a MAC/Apple system, in addition to the processor? Just trying to future-proof this, if we add ARM support in Linux as well.

Reply:
I believe `if(CMAKE_SYSTEM_NAME MATCHES "Darwin")` should do it. I'll split that if statement and set it to `NATIVE` for Linux for now.